ranger-core/src/test/java/com/flipkart/ranger/core/TestUtils.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2015 Flipkart Internet Pvt. Ltd.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.flipkart.ranger.core;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.awaitility.Awaitility.await;
+
+/**
+ *
+ */
+public class TestUtils {
+    public static void sleepForSeconds(int numSeconds) {
```

Comment:
This function was kind of a temporary hack. This gives us a _lot_ of grief in CI builds. Ideally we should use awaility and wait with max bounds till a condition becomes true. Not sure how invovled the work would be, but it would be an excellent upgrade in testability to ger rid of this POS function

Reply:
I will look into this in a day or two and submit an MR.